main/src/main/scala/org/clulab/numeric/actions/NumericActions.scala

```diff
@@ -265,7 +265,7 @@ class NumericActions(seasonNormalizer: SeasonNormalizer, unitNormalizer: UnitNor
     val (seasonMentions, otherMentions) = mentions.partition(m => m.foundBy.contains("season"))
     val (springFall, otherSeasons) = seasonMentions.partition(m => m.text.equalsIgnoreCase("spring") || m.text.equalsIgnoreCase("fall"))
     val trueSeasons = springFall.filter { m =>
-      m.tags.head.contains("NN") && {
+      m.tags.get.head.startsWith("NN") && {
```

Comment:
Thank you!

This encourages me to continue denoting Options by name (like tagsOpt) and wonder whether something like the [TagSet](https://github.com/clulab/eidos/blob/master/eidoscommon/src/main/scala/org/clulab/wm/eidoscommon/TagSet.scala) class from Eidos should be ported.  We probably shouldn't be using string operations for classifying tags.